src/overlays/submenu/submenu_ovl/m_submenu_ovl.c

```diff
@@ -602,11 +602,23 @@ s32 add_trigger_954[] = { R_CBUTTONS, U_CBUTTONS, L_CBUTTONS, D_CBUTTONS };
 
 f32 move_data_1027[2][4] = { { 2.0f, 0.0f, 300.0f, 1.0f }, { 0.5f, 120.0f, 0.0f, -1.0f } };
 
+#define OPEN_CUSTOM_POLY_OPA()                \
+    {                                         \
+        Gfx* __polyOpa = __gfxCtx->polyOpa.p; \
+        int __opa_opened = 0;                 \
+        do {                                  \
+        } while (0)
+
+#define CLOSE_CUSTOM_POLY_OPA()      \
+    __gfxCtx->polyOpa.p = __polyOpa; \
+    (void)__opa_opened;              \
+    }                                \
+    do {                             \
+    } while (0)
+
```

Comment:
Perhaps just me, but I wonder if we should add an `#define CUSTOM_POLY_OPA __polyOpa` for the access of the `polyOpa` in these macros?

Reply:
When we first were using this macro, emily proposed just using `__polyOpa` directly here: https://discord.com/channels/688807550715560050/961807933564403752/1174036188986880010

I do think we'll want a macro here at some point. But I'm not sure if we want one right this moment.